editor/plugins/animation_blend_space_1d_editor.cpp

```diff
@@ -473,7 +479,7 @@ void AnimationNodeBlendSpace1DEditor::_update_edited_point_pos() {
 
 void AnimationNodeBlendSpace1DEditor::_update_tool_erase() {
 	bool point_valid = selected_point >= 0 && selected_point < blend_space->get_blend_point_count();
-	tool_erase->set_disabled(!point_valid);
+	tool_erase->set_disabled(!point_valid || blend_space_is_foreign);
```

Comment:
I would just call it read_only, as I mentioned in other PRs, we also want this for locked resources for VCS eventually.